api/server_test.go

```diff
@@ -121,7 +121,7 @@ func TestServerRoutes(t *testing.T) {
 		{"GET", "/profile", "", "profileResponse.json", 200},
 		{"GET", "/me", "", "profileResponse.json", 200},
 
-		{"POST", "/add", "addRequestFromURL.json", "addResponseFromURL.json", 200},
+		{"POST", "/new", "addRequestFromURL.json", "addResponseFromURL.json", 200},
```

Comment:
Let's also rename the test files to `newRequestFromURL.json` and `newResponseFromURL.json`.